### ISSUE TITLE
False positive for WPS226 with string types

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1127]
I get this error when checking `returns` source code:

```
./returns/context/requires_context_result.py
  0:1      WPS226 Found string constant over-use: RequiresContextResult[_EnvType, _NewValueType, _ErrorType]
```

It is really ugly that our linter checks string types as constants.
They should be excluded from the check.

There's nothing I can do about them.

**Comment by sobolevn** [comment]
Link: https://github.com/dry-python/returns/blob/master/returns/context/requires_context_result.py
